### DISCUSSION TITLE
How to change color in custom theme [vuetify 3.0.0-beta.15][vue 3.2.37]

### DISCUSSION THREAD

**Discussion by AniaKru95** [vuetifyjs/vuetify#16045] [Q&A] [answered]
I would like to change success color in my custom theme. 
Primary and secondary colors overwrite but success color doesn't.

**My Theme:**
```
import {createVuetify, ThemeDefinition} from 'vuetify'
import {mdi} from "vuetify/iconsets/mdi";
import {aliases, fa} from 'vuetify/iconsets/fa-svg'
import '@mdi/font/css/materialdesignicons.css'
import 'vuetify/styles'
import {Resize} from 'vuetify/directives'

const myTheme: ThemeDefinition = {
    dark: false,
    colors: {
        background: '#F5F5F5FF',
        surface: '#FFFFFF',
        primary: '#657686',
        secondary: '#10E49C',
        error: '#F14668',
        info: '#3E8ED0',
        success: '#48C78EFF',
        warning: '#ffdd80'
    }
}


export default createVuetify({
    directives: {Resize},
    icons: {
        defaultSet: 'fa',
        aliases,
        sets: {
            fa,
            mdi,
        },
    },
    theme: {
        defaultTheme: 'myTheme',
        themes: {
            myTheme
        }
    }
})
```

**I use success color in snack bar:** 
```
    <v-snackbar
        absolute
        location="top"
        variant="tonal"
        vertical
        v-model="snackbar"
        :timeout="3000"
        color="success">
      Test message
    </v-snackbar>
```

This is my snackbar with test message and another color:
![image](https://user-images.githubusercontent.com/84393751/200545214-d77503c1-e83b-43a9-99e3-f2ae94d5def9.png)

![image](https://user-images.githubusercontent.com/84393751/200545420-4033ffa8-4174-416a-853d-80a53f2d0454.png)

![image](https://user-images.githubusercontent.com/84393751/200545530-692e49b4-74c1-4277-b9f3-579f46219b91.png)

I have no idea, where does this color come from. I do not declaring it anywhere.

Please for your help.

**Comment by KaelWD** [comment, marked as answer]
Theme colors don't support transparency, it's being parsed as `#C78EFF`

@nekosaur #15667 removed the warning for this, but `colorToRGB` only uses the 3 smallest bytes. Just checking for `> 0xffffff` wouldn't work either because `#00123456` and `#123456` are both `0x123456` even though the first is `#001234` with transparency. This also isn't symmetric: `colorToHex('#00123456') === '#123456'`

**Reply by AniaKru95** [reply]
Oh, finally I understand, thank you
It's help me a lot! :)
